pkg/client/operatorclient.go

```diff
@@ -21,6 +22,17 @@ type OperatorHelperClient struct {
 	client   operatorclientv1.NetworkInterface
 }
 
+// ApplyOperatorSpec and ApplyOperatorStatus were added here:
+// https://github.com/openshift/library-go/commit/2e06abc0a54d9ecd0a8783325068896c347bd137#diff-0e7fb0f95ea48e43d0cad6743df9338e45348ebb2ca9aa0c5491eda440642b45
+// added as no-ops for now.
+// TO-DO: investigate if it's better to implement and use these.
```

Comment:
I think they get called from the library code even if you don't call them directly...

Reply:
@danwinship , are you saying I should find a way to implement these better then?